modules/bitgo/test/browser/browser.spec.ts

```diff
@@ -33,8 +33,6 @@ describe('Coins', () => {
       XrpToken: 1,
       Rune: 1,
       Trune: 1,
-      Apt: 1,
```

Comment:
you will be adding these at the time implementation of apt sdk?

Reply:
Added this in this PR: https://github.com/BitGo/BitGoJS/pull/5115/files#diff-f8b260fd3dd9567a3a5c8d55b54d988b7428d1d7f1fb7c9f7e5ede728a4a94faR36
As the browser test case was failing without statics, so had to put this coin in excluded list.